os/arch/arm/src/s5j/s5j_pwm.c

```diff
@@ -432,7 +432,9 @@ FAR struct pwm_lowerhalf_s *s5j_pwminitialize(int timer)
 		lldbg("ERROR: invalid PWM is requested\n");
 	}
 
-	s5j_pwm_reset(lower, timer);
+	if (lower) {
```

Comment:
How about add "return NULL;" instead of this?
Isn't it better than checking conditional for perfermance?
```
if (xx) {
...
} else {
  lldbg
  return NULL;
}

s5j_pwm_reset

return
```